Gemfile.lock

```diff
@@ -207,7 +207,7 @@ GEM
     nokogiri (1.12.5)
       mini_portile2 (~> 2.6.1)
       racc (~> 1.4)
-    nokogiri (1.12.5-x86_64-darwin)
+    nokogiri (1.12.5-arm64-darwin)
```

Comment:
This prevents `bundle install` from working on Intel Macs (with built-in Ruby) 😬

Please undo!

Reply:
Hmm, nasty.  I wonder how it decided to do that when I ran it on Intel myself - must not be a factor of the platform.

Reply:
FWIW I see the same issue when running it on my machine - it seems like a bug in the version of Ruby preinstalled on macOS :-/